includes/class-sensei-utils.php

```diff
@@ -127,17 +127,17 @@ public static function sensei_check_for_activity( $args = array(), $return_comme
 			$args['status'] = 'any';
 		}
 
-		// Get the comments
 		/**
 		 * This filter runs inside Sensei_Utils::sensei_check_for_activity
 		 *
 		 * It runs while getting the comments for the given request.
 		 *
 		 * @param int|array $comments
+		 * @param array $args Search arguments.
 		 */
-		$comments = apply_filters( 'sensei_check_for_activity', get_comments( $args ) );
+		$comments = apply_filters( 'sensei_check_for_activity', get_comments( $args ), $args );
```

Comment:
Should we add a `Hooks` label for this one?

Reply:
Yup, adding!